scripts/eurostat/life_expectancy_by_age_gender/preprocess.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+import re
+import sys
+path = 'demo_r_mlifexp.tsv'
+
+def group_to_wide_form(data, col):
+    """reorgnize the data of each statvars into independent columns """
```

Comment:
Could you flush out the docstring or add more comments? Having some trouble understanding this func.

Reply:
Added more description.